autoload/multiple_cursors.vim

```diff
@@ -268,6 +273,7 @@ function! multiple_cursors#find(start, end, pattern)
     echohl ErrorMsg | echo 'No match found' | echohl None
     return
   else
+    call s:fire_pre_triggers()
```

Comment:
Oh this must have just been missed when we switched to autocommands, nice catch.

If you note the code block a couple of lines below which calls the old `Multiple_cursors_before`, the call to `fire_pre_triggers` should probably replace that instead of going up here.